packages/build-info/src/frameworks/framework.ts

```diff
@@ -56,6 +56,7 @@ export interface Framework {
     command: string
     port?: number
     pollingStrategies?: PollingStrategy[]
+    clearDist?: boolean
```

Comment:
For clarification, what is `Dist`? is it always a `dist` directory, or is it some output directory that can be configured, like the `publish` one?

Reply:
In the case of Nuxt, it's `dist` - but more generally, it's the publish directory. Let me rename this!

Reply:
Done in https://github.com/netlify/build/pull/5430/commits/d2ebd2b132489b414a1987ece2528aaba76c71e6